nextcloudappstore/core/caching.py

```diff
@@ -33,6 +33,10 @@ def app_rating_etag(request: Any, id: str) -> str:
     return create_etag([(AppRating.objects.filter(app__id=id), 'rated_at')])
 
 
+def all_apps_etag(request: Any) -> str:
+    return create_etag([(App.objects.all(), 'last_modified')])
```

Comment:
Would also include the app release delete log like in apps_etag. In fact you should be able to reuse the function without changes

Reply:
Updated. Is this how you imagined it?